zetaclient/chains/evm/signer/gas.go

```diff
@@ -71,18 +70,21 @@ func gasFromCCTX(cctx *types.CrossChainTx, logger zerolog.Logger) (Gas, error) {
 		limit  = params.CallOptions.GasLimit
 	)
 
+	isWithdrawWithNoCall := !cctx.IsCurrentOutboundRevert() && cctx.InboundParams.IsCrossChainCall
+	isRevertWithNoCall := cctx.IsCurrentOutboundRevert() && cctx.RevertOptions.CallOnRevert
```

Comment:
Question: `CallOnRevert == true` means a `NoCall` or not?

Had this dump question because the meaning of the prefix `NoCall` in the name `isRevertWithNoCall` confuses me a bit.